shenyu-web/src/main/java/org/apache/shenyu/web/configuration/properties/CrosFilterProperties.java

```diff
@@ -0,0 +1,168 @@
+package org.apache.shenyu.web.configuration.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * CrosFilterProperties
+ *
+ */
+@Component
+@ConfigurationProperties(prefix = "shenyu.cros")
+public class CrosFilterProperties {
```

Comment:
CrosFilterProperties maybe in ShenyuConfig.java ,not this  @ConfigurationProperties

Reply:
yes, I pull the new code from master.